### PR TITLE
Changed theme.css file

### PR DESCRIPTION
**Pull request by sveneschlbeck**
#4803

**What kind of change does this PR introduce?** (check at least one)

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [x ] Other, please describe: Added panel info to ``theme.css``

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [ x] No

### CODE REVIEW

**Review by ArchBlood — CHANGES_REQUESTED** [review]
The changes here just add the CSS to the bottom of the file, it should be placed where the other `.panel` entries are, not only this but it only adds parts of what #4803 is interpreting.

**Comment by luke-** [comment]
Thank you for your contribution.
The `theme.css` file is automatically compiled from the various LESS files and therefore cannot be edited manually.

The following file is used for this purpose:
https://github.com/humhub/humhub/blob/develop/themes/HumHub/less/build.less

**Comment by sveneschlbeck** [comment]
@luke- Do you mean this file?

https://github.com/humhub/humhub/blob/develop/themes/HumHub/less/theme.less

**Comment by sveneschlbeck** [comment]
Is the ``.panel-info`` supposed to just be added to the so-far empty LESS file?

**Comment by felixhahnweilheim** [comment]
https://github.com/humhub/humhub/blob/master/static/less/panel.less

File: static/less/panel.less [hunk 1]
```diff
@@ -206,6 +206,13 @@
     }
 }
 
+.panel-info{
+    border:2px solid #5bc0de
+}
+.panel-info .panel-heading{
+    color:#5bc0de
```
Comment (luke-) [inline static/less/panel.less:213]: Please do not use hardcoded color values. Instead use color variables like `color:@info` 

You can find available color variables here:
https://github.com/humhub/humhub/blob/master/static/less/variables.less#L12

**Review by luke- — CHANGES_REQUESTED** [review]

File: static/less/panel.less (hunk 1 above)
Comment (sveneschlbeck) [reply static/less/panel.less]: Thanks! Done

File: static/less/panel.less [hunk 2]
```diff
@@ -206,6 +206,13 @@
     }
 }
 
+.panel-info{
```
Comment (luke-) [inline static/less/panel.less:209]: Please add a whitespace between classname and bracket.

File: static/less/panel.less [hunk 3]
```diff
@@ -206,6 +206,13 @@
...
+    border:2px solid #5bc0de
```
Comment (luke-) [inline static/less/panel.less:210]: Please also use color code here and end the line with semicolon.

File: static/less/panel.less [hunk 4]
```diff
@@ -206,6 +206,13 @@
...
+}
+.panel-info .panel-heading{
+    color: @info
```
Comment (luke-) [inline static/less/panel.less:213]: Please end the line with semicolon.

File: static/less/panel.less [hunk 5]
```diff
@@ -206,6 +206,13 @@
     }
 }
 
+.panel-info{
+    border:2px solid #5bc0de
+}
+.panel-info .panel-heading{
```
Comment (luke-) [inline static/less/panel.less:212]: Please add a whitespace between classname and bracket.

**Review by luke- — CHANGES_REQUESTED** [review]
Regarding the

**Review by ArchBlood — APPROVED** [review]
Looks good to me

**Review by luke- — APPROVED** [review]

**Pull request merged**